languages/cpp/exercises/concept/numbers/numbers.cpp

```diff
@@ -0,0 +1,3 @@
+#include "numbers.h"
+
+namespace assembly_line {}
```

Comment:
The stub file does not contain stubs for the methods the student needs to implement. These should be added (see [the docs](https://github.com/exercism/v3/blob/master/docs/concept-exercises.md#stub-implementation-file)).

Reply:
Added